src/GitHub.VisualStudio/Services/GitHubServiceProvider.cs

```diff
@@ -97,16 +97,23 @@ public GitHubServiceProvider(IServiceProviderPackage asyncServiceProvider, IServ
 
         public object TryGetService(Type serviceType)
         {
-            string contract = AttributedModelServices.GetContractName(serviceType);
+            var contract = AttributedModelServices.GetContractName(serviceType);
             var instance = AddToDisposables(TempContainer.GetExportedValueOrDefault<object>(contract));
             if (instance != null)
                 return instance;
 
             var sp = initialized ? syncServiceProvider : asyncServiceProvider;
 
-            instance = sp.GetService(serviceType);
-            if (instance != null)
-                return instance;
+            try
+            {
+                instance = sp.GetService(serviceType);
+                if (instance != null)
+                    return instance;
+            }
+            catch (Exception ex)
```

Comment:
This seems unrelated to trace logging. No objection to including it here, but could you explain how it got here? :D

Reply:
I'm unable to reproduce the error @shana and I saw that caused us to add this catch.

Reply:
Should I remove the try/catch?

Reply:
No, leave it in, it makes sense to log exceptions here. I just wondered what caused you to add it.